src/lib/utils/__tests__/index-test.js

```diff
@@ -1165,3 +1166,16 @@ describe('utils.getPropertyByPath', () => {
     expect(utils.getPropertyByPath(object, 'nested.name')).toBe(undefined);
   });
 });
+
+describe('utils.withAbsolutePosition', () => {
```

Comment:
For new utils we can create specific test files (e.g. `utils/__tests__/hits-absolute-position-test.js`).